provision/minikube/keycloak/config/kcb-infinispan-cache-remote-store-config.xml

```diff
@@ -0,0 +1,261 @@
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<infinispan
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="urn:infinispan:config:14.0 https://www.infinispan.org/schemas/infinispan-config-14.0.xsd
+            urn:infinispan:config:store:remote:14.0 https://www.infinispan.org/schemas/infinispan-cachestore-remote-config-14.0.xsd"
+        xmlns="urn:infinispan:config:14.0">
+
+    <!-- the statistics="true" attribute is not part of the original KC config and was added by Keycloak Benchmark -->
+    <cache-container name="keycloak" statistics="true">
+        <transport lock-timeout="60000"/>
+        <metrics names-as-tags="true" />
+        <local-cache name="realms" simple-cache="true" statistics="true">
+            <encoding>
+                <key media-type="application/x-java-object"/>
+                <value media-type="application/x-java-object"/>
+            </encoding>
+            <memory max-count="10000"/>
+        </local-cache>
+        <local-cache name="users" simple-cache="true" statistics="true">
+            <encoding>
+                <key media-type="application/x-java-object"/>
+                <value media-type="application/x-java-object"/>
+            </encoding>
+            <memory max-count="10000"/>
+        </local-cache>
+        <distributed-cache name="sessions" owners="2" statistics="true">
+            <expiration lifespan="-1"/>
+            <persistence passivation="false">
+                <remote-store xmlns="urn:infinispan:config:store:remote:14.0"
+                              cache="sessions"
+                              raw-values="true"
+                              shared="true"
+                              segmented="false">
+                    <remote-server host="${env.KC_REMOTE_STORE_HOST}"
+                                   port="${env.KC_REMOTE_STORE_PORT}"/>
+                    <connection-pool max-active="16"
+                                     exhausted-action="CREATE_NEW"/>
+                    <security>
+                        <authentication server-name="infinispan">
+                            <digest username="${env.KC_REMOTE_STORE_USERNAME}"
+                                    password="${env.KC_REMOTE_STORE_PASSWORD}"
+                                    realm="default"/>
+                        </authentication>
+                        <encryption protocol="TLSv1.3"
+                                    sni-hostname="${env.KC_REMOTE_STORE_HOST}">
+                            <truststore filename="/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
+                                        type="pem"/>
+                        </encryption>
+                    </security>
+                </remote-store>
+            </persistence>
+            <state-transfer enabled="false"/>
+            <state-transfer enabled="false"/>
```

Comment:
`<state-transfer enabled="false">` tag is written twice.

Reply:
Thank you - updated the PR.